### PR TITLE
[roaring] Update to 0.9.6

### PR DESCRIPTION
**Pull request by yuyoyuppe**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/roaring/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "roaring",
-  "version": "0.7.1",
+  "version": "0.9.5",
   "port-version": 1,
```
Comment (jimwang118) [inline ports/roaring/vcpkg.json:4]:
```suggestion

```

File: ports/roaring/portfile.cmake [hunk 2]
```diff
@@ -5,8 +5,8 @@ endif ()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO RoaringBitmap/CRoaring
-    REF v0.7.1
-    SHA512 1d49da4e5b5ed0726feaf3608488e24f13c31358d1a72c7d0086fb6210313f73164745e0d1b616e8930073bbe259f7e1ac09fc97473b659694248f627c762f02
+    REF v0.9.5
```
Comment (jimwang118) [inline ports/roaring/portfile.cmake:8]:
```suggestion
    REF "v{VERSION}"

```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

File: ports/roaring/vcpkg.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "roaring",
-  "version": "0.7.1",
+  "version": "0.9.5",
```
Comment (jimwang118) [inline ports/roaring/vcpkg.json:3]:
```suggestion
  "version": "0.9.6",
```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by yuyoyuppe** [comment]
@jimwang118 updated, PTAL.

**Review by jimwang118 — APPROVED** [review]

**Comment by jimwang118** [comment]
All test passed with following triplets:

x86-windows
x64-windows
x64-windows-static

**Review by JonLiu1993 — APPROVED** [review]

**Review by jimwang118 — APPROVED** [review]

**Pull request merged**